packages/suite/src/components/suite/ChangeDeviceLabel.tsx

```diff
@@ -96,25 +99,26 @@ export const ChangeDeviceLabel = ({
         isDeviceLocked || (!placeholder && label === device?.label) || !!error || !label;
 
     return (
-        <>
+        <Container>
```

Comment:
no need for this, I believe. Just make the button `isFullWidth` :)

Reply:
It doesn't work for Input

Reply:
It should be fixed in f22e3ec86d701e77122fe754ddea03cb34c118ab

I'm pretty sure that the design will be changed here.

Reply:
This is what i meant: 2704e14b42

Also changing the `size` of both so it looks nicer ;)

![image](https://github.com/trezor/trezor-suite/assets/45338719/fc68f174-9fb2-495c-ad3d-a58e8b644d65)